test/index.js

```diff
@@ -339,6 +339,38 @@ describe('Likely', function () {
         });
     });
 
+    describe('updated URL for sharing after changing the data-url on the container [#151]', function () {
+        const targetUrl = 'ya.ru';
+
+        before(function () {
+            return getLikelyPage(driver, LikelyPage.ISSUE_151)
+                .then(() => waitUntilLikelyInitialized(driver))
+                .then(() => {
+                    return driver.executeScript(`
+                        document.querySelector('.likely').setAttribute('data-url', 'https://${targetUrl}/');
+                    `);
+                });
+        });
+
+
+        const testedServices = [
+            { name: 'Facebook', likelyName: 'facebook', urlRegex: `${encodeURIComponent(targetUrl)}` },
+            { name: 'Odnoklassniki', likelyName: 'odnoklassniki', urlRegex: `${encodeURIComponent(targetUrl)}` },
+            { name: 'Telegram', likelyName: 'telegram', urlRegex: `${encodeURIComponent(targetUrl)}` },
+            { name: 'Twitter', likelyName: 'twitter', urlRegex: `${encodeURIComponent(targetUrl)}` },
+            { name: 'VK', likelyName: 'vkontakte', urlRegex: `${encodeURIComponent(targetUrl)}` },
+            // Pinterest and Linkedin do not contain sharing URLs if not authorized and the test will fail
```

Comment:
Just out of curiosity, wouldn't this happen for these buttons in production?

Reply:
This is a problem only for this particular auto test. We cannot determine that the data has been verified. It's not a problem.